servicebroker/lib/util/apiServerUtil.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var request = require('request');
+
+module.exports = function(apiServerUri){
+  var apiUtil = {};
+  apiServerUri = apiServerUri.replace(/\/$/g, "").toLowerCase();
+  apiUtil.attachPolicy= function(appId, policy, callback) {
+    var options = {
+      uri: apiServerUri + '/v1/apps/' + appId + '/policy',
+      method: 'PUT',
+      json: policy
+    };
+    request(options, function(error, response, body) {
```

Comment:
is this better than request.put({uri: foo, json: policy}, function(err, rsp, body) {
})?